Tests/Marketplace/packs_dependencies.py

```diff
@@ -27,6 +30,50 @@ def option_handler():
     return parser.parse_args()
 
 
+@contextmanager
+def ProcessPoolHandler() -> ProcessPool:
+    """ Process pool Handler which terminate all processes in case of Exception.
+
+    Yields:
+        ProcessPool: Pebble process pool.
+    """
+    with ProcessPool(max_workers=3) as pool:
+        try:
+            yield pool
+        except KeyboardInterrupt:
```

Comment:
Why is this case needed?
this is not sdk and the behavior is the same as every exception

Reply:
Done, Correct